diff/parse.go

```diff
@@ -324,7 +354,7 @@ func (r *FileDiffReader) ReadExtendedHeaders() ([]string, error) {
 				return xheaders, OverflowError(line)
 			}
 		}
-		if bytes.HasPrefix(line, []byte("--- ")) {
+		if bytes.HasPrefix(line, []byte("--- ")) || (bytes.HasPrefix(line, onlyInMessagePrefix)) {
```

Comment:
Similar to above. Is it not allowed to have an extended header line starting with "Only in "? Maybe even an extended header before the actual "Only in " line?